src/server/webkit/wkPage.ts

```diff
@@ -1043,7 +1043,11 @@ export class WKPage implements PageDelegate {
         validFrom: responseReceivedPayload?.response.security?.certificate?.validFrom,
         validTo: responseReceivedPayload?.response.security?.certificate?.validUntil,
       });
-      response._requestFinished(helper.secondsToRoundishMillis(event.timestamp - request._timestamp));
+      const { responseBodyBytesReceived, responseHeaderBytesReceived } = event.metrics || {};
+      const transferSize = responseBodyBytesReceived && responseHeaderBytesReceived ? responseBodyBytesReceived + responseHeaderBytesReceived : undefined;
```

Comment:
It doesn't seem to account for compression, is it intentional?

Reply:
Yes afaik, but let me verify it later against DevTools.

Reply:
I believe it should be equal to responseHeaderBytesReceived if the response has empty body.